protocol/contracts/interfaces/oud/Oud__ITempleERC20Token.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.8.4;
```

Comment:
Any reason why the version of solidity is different for this one?

Reply:
copy of the interface from elsewhere. thought should preserve?

Reply:
source > https://github.com/TempleDAO/temple/blob/stage/protocol/contracts/amo/interfaces/AMO__ITempleERC20Token.sol